lib/publiccloud/provider.pm

```diff
@@ -363,15 +366,16 @@ sub terraform_apply {
     $args{count} //= '1';
     my $instance_type = get_var('PUBLIC_CLOUD_INSTANCE_TYPE');
     my $image = $self->get_image_id();
+    my $image_uri = get_var("PUBLIC_CLOUD_IMAGE_URI");
     my $ssh_private_key_file = '/root/.ssh/id_rsa';
     my $cloud_name = $self->conv_openqa_tf_name;
 
     record_info('WARNING', 'Terraform apply has been run previously.') if ($self->terraform_applied);
 
     $self->terraform_prepare_env();
 
-    record_info('INFO', "Creating instance $instance_type from $image ...");
     if (get_var('PUBLIC_CLOUD_SLES4SAP')) {
+        record_info('INFO', "Creating instance $instance_type from $image ...");
```

Comment:
I meant something different . This message is relevant not only for SAP and should stay outside this if in old place . Instead when `$image` variable is empty ( and AFAIK it will be empty only when `$image_uri` is defined we need to show "Creating instance $instance_type from $image_uri ..."